config/initializers/rack_attack.rb

```diff
@@ -0,0 +1,15 @@
+
+# Rack attack protection
+module Rack
+  class Attack
+    throttle('req/ip', limit: 120, period: 1.minute, &:ip)
+
+    blocklist('auth-less annoyances') do |req|
```

Comment:
I'm not sure this is applicable. `Keystorm` will have to process requests without tokens ... because its purpose is to issue tokens :-) Also, I haven't seen the `Tokenator` class anywhere in the pull request or `master`.